### PR TITLE
DLS-9829 Update bootstrap

### PR DESCRIPTION
**Pull request by LAJW**

### CODE REVIEW

**Comment by LAJW** [comment]
Stub generator can't be removed from the test scope as it's used to generate the stub.

File: test/resources/logback-test.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+
+    <appender name="FILE" class="ch.qos.logback.core.FileAppender">
+        <file>logs/help-to-save.log</file>
```
Comment (johno1985) [inline test/resources/logback-test.xml:5]: log file name needs correcting

**Review by johno1985 — CHANGES_REQUESTED** [review]
test log file name needs correcting

File: test/resources/logback-test.xml (hunk 1 above)
Comment (LAJW) [reply test/resources/logback-test.xml]: Addressed.

**Review by johno1985 — APPROVED** [review]

**Pull request merged**